packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -817,6 +829,11 @@ export class RunningSummarizer extends TypedEventEmitter<ISummarizerEvents> impl
 
 		// If summarization is still unsuccessful, stop the summarizer.
 		if (status === "failure") {
+			this.mc.logger.sendErrorEvent({
+				eventName: "SummarizeFailed",
+				maxAttempts,
+				summaryAttempts: currentAttempt,
```

Comment:
This event does not contain the error that caused the failure or other details that summarize_cancel has. Is the idea to look at the summarize_cancel event if you need that information? Can we include that information here? It will be the details for the last attempt of course.